client/post-editor/media-modal/index.jsx

```diff
@@ -109,24 +121,19 @@ export class EditorMediaModal extends Component {
 		};
 	}
 
-	isDisabled() {
-		return some( this.props.mediaLibrarySelectedItems, function( item ) {
-			const mimePrefix = MediaUtils.getMimePrefix( item );
-			return item.transient && ( mimePrefix !== 'image' || ModalViews.GALLERY === this.props.view );
-		}.bind( this ) );
-	}
-
 	confirmSelection = () => {
 		const { view, mediaLibrarySelectedItems } = this.props;
 
-		let value;
-		if ( mediaLibrarySelectedItems.length ) {
-			value = {
+		if ( areMediaActionsDisabled( view, mediaLibrarySelectedItems ) ) {
```

Comment:
Why did we need to change this behavior? When would `confirmSelection` be called while `areMediaActionsDisabled` is `true` ? Even if it would, is the ideal behavior that the modal not be closed?